lasagne/layers/prelu.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import theano.tensor as T
+
+from .. import init
+from .. import nonlinearities
+
+from .base import Layer
+
+__all__ = [
+    "PReLULayer"
+]
+
+class PReLULayer(Layer):
+    """
+    A layer used to implement Parametric Rectified Linear Units (PReLU).
+    This layer simply applies the parametric nonlinearity to the previous
+    layer's output. 
+
+    Parameters
+    -----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape.
+
+    alpha : double or None
```

Comment:
I think `float` is more appropriate as a type name. But we haven't really standardized this across the docs yet. We probably should.